tests_integration/nativeapp/test_open.py

```diff
@@ -79,15 +38,12 @@ def test_nativeapp_open_v2(
     project_name = "myapp"
     app_name = f"{project_name}_{USER_NAME}"
 
-    # TODO Use the main project_directory block once "snow app run" supports definition v2
-    with project_directory("napp_init_v1"):
+    with project_directory(f"napp_init_{definition_version}"):
```

Comment:
Do we use `definition_version` other than to interpolate in this string? If not, we might want to parametrize the project directory name instead (similar to other tests)

Reply:
Good suggestion! Simplified it here and across all other tests